site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -160,34 +174,143 @@ Response:
 ```
+### Metadata Management
 
-### Reserve
+#### Set Label
 
-Move Gameserver into a Reserved state for a certain amount of seconds for the future allocation.
+Apply a Label with the prefix "agones.dev/sdk-" to the backing `GameServer` metadata. 
 
-- Path: `/reserve`
-- Method: `POST`
-- Body: `{"seconds": "5"}`
+See the SDK [SetLabel]({{< ref "/docs/Guides/Client SDKs/_index.md#setlabel-key-value" >}}) documentation for restrictions.
 
-#### Example
+##### Example
 
 ```bash
-$ curl -d '{"seconds": "5"}' -H "Content-Type: application/json" -X POST http://localhost:${AGONES_SDK_HTTP_PORT}/reserve
+$ curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/label
 ```
 
-### Allocate
+#### Set Annotation
 
-With some matchmakers and game matching strategies, it can be important for game servers to mark themselves as `Allocated`.
-For those scenarios, this SDK functionality exists. 
+Apply a Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer` metadata
 
-{{< alert title="Note" color="info">}}
-Using a [GameServerAllocation]({{< ref "/docs/Reference/gameserverallocation.md" >}}) is preferred in all other scenarios, 
-as it gives Agones control over how packed `GameServers` are scheduled within a cluster, whereas with `Allocate()` you
-relinquish control to an external service which likely doesn't have as much information as Agones.
-{{< /alert >}}
+##### Example
+
+```bash
+$ curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
+```
+
+{{% feature publishVersion="1.6.0" %}}
+### Player Tracking
 
-#### Example
+{{< alpha title="Player Tracking" gate="PlayerTracking" >}}
+
+#### Alpha: PlayerConnect
+
+This function increases the SDK’s stored player count by one, and appends this playerID to 
+`GameServer.Status.Players.IDs`.
+    
+##### Example    
 
 ```bash
-$ curl -d "{}" -H "Content-Type: application/json" -X POST http://localhost:${AGONES_SDK_HTTP_PORT}/allocate
+$ curl -d '{"playerID": "uzh7i"}' -H "Content-Type: application/json" -X POST http://localhost:${AGONES_SDK_HTTP_PORT}/alpha/player/connect
+```
+
+Response:
+```json
+{"bool":true}
 ```
+    
+#### Alpha: PlayerDisconnect
+
+This function decreases the SDK’s stored player count by one, and removes the playerID from 
+[`GameServer.Status.Players.IDs`][playerstatus].
+
+##### Example
+
+```bash
+$ curl -d '{"playerID": "uzh7i"}' -H "Content-Type: application/json" -X POST http://localhost:${AGONES_SDK_HTTP_PORT}/alpha/player/disconnect
+```
+
+Response:
+```json
+{"bool":true}
+```
+
+#### Alpha: SetPlayerCapacity
+
+Update the [`GameServer.Status.Players.Capacity`][playerstatus] value with a new capacity.
```

Comment:
It might be worth clarifying that this capacity is not enforced as that is not a responsibility of Agones

Reply:
This is a good idea, I'm going to tackle this in #1569 , since it doesn't seem REST API specific.